### DISCUSSION TITLE
Question : Clean our aws accounts from policies deployed as a lambda

### DISCUSSION THREAD

**Discussion by mikrammo** [cloud-custodian/cloud-custodian#7350] [General]
Hello all,
I'm testing deploying cloudcustodian policies as a lambda, in multi-account using c7n-org.
My question is how we can manage lifecyle of the policies deployed as a lambda, and how to delete them if we decide that this policy is no longer useful for us

**Comment by kapilt** [comment]
see tools/ops/mugc.py it will delete lambdas that are no longer defined in a set of policy files, it can be run across accounts with c7n-org run-script

**Comment by mikrammo** [comment]
Thanks for your answer, I'll try it.
From this docs, I didn't see mu project, https://cloudcustodian.io/docs/
But on google I founded https://cloudcustodian.io/docs/policy/mu.html
It's a good tool, it needs to be better documented

**Comment by kapilt** [comment]
yeah.. leaving this open till we add some docs for it, afaics the tool isn't in the docs, though its commonly referenced in our chat.
